.github/workflows/build-and-deploy-test.yml

```diff
@@ -34,5 +34,5 @@ jobs:
     - name: Publish distribution to Test PyPI
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
-        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        password: ${{ secrets.TEST_TOOLS_PYPI_PAK }}
```

Comment:
I don't see a `TEST_TOOLS_PYPI_PAK` secret available in this repository or the organization, should one be created? Is this workflow necessary? I don't recall our other python packages bothering to publish to test.pypi.org